integrations/crowdnode/src/main/java/org/dash/wallet/integrations/crowdnode/ui/dialogs/StakingDialog.kt

```diff
@@ -37,9 +43,17 @@ class StakingDialog: OffsetDialogFragment() {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
-        binding.stakingMessage.text = getString(
+        binding.stakingMessageTwo.text = getString(
+            R.string.crowdnode_staking_info_message_second,
+            String.format(Locale.getDefault(), "%.1f", viewModel.getMasternodeAPY())
+        )
+        binding.stakingFirstMinDepositMessage.text = getString(
             R.string.crowdnode_staking_info_message,
             CrowdNodeConstants.DASH_FORMAT.format(CrowdNodeConstants.MINIMUM_DASH_DEPOSIT)
         )
+        binding.stakingApyTitle.text = getString(
+            R.string.crowdnode_staking_apy_title,
+            String.format(Locale.getDefault(), "%.1f", viewModel.getCrowdNodeAPY())
+        )
```

Comment:
not using livedata for these values.  we can't get inside CrowdNode unless blockchain is synced and the APY doesn't change often.